libhb/hb.c

```diff
@@ -1609,6 +1609,35 @@ int hb_add( hb_handle_t * h, hb_job_t * job )
     return job_copy->sequence_id;
 }
 
+void hb_job_setup_pix_ftm(hb_job_t * job)
+{
+    int bit_depth = hb_get_bit_depth(job->title->pix_fmt);
```

Comment:
Can this ever return > 10

Reply:
Yes, it's less common but there are format with an higher bit depth.
The only 12bit encoder is x265, so this should be expanded to 12bit too.

Reply:
yup. Was going to suggest

if (bit_depth == 10  should be >= below as a starting point.